.idea/WANACODE-credit-card-task.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remember the video about git? There was a .gitignore file described. And what was listed there was `.idea` directory. It was done to prevent commiting `.idea` directory, as it is not needed for the project to run. Please follow the guidlines given in the video :)